File: backend/src/main/java/com/andrew/hdss/dtos/VisitPushDto.java
package com.andrew.hdss.dtos;// package: match your existing dto package

import com.andrew.hdss.models.enums.VisitStatus;

import java.time.Instant;
import java.time.LocalDateTime;

public record VisitPushDto(
        String clientId,
        String householdClientId,
        LocalDateTime visitDate,
        VisitStatus status
) {}

File: android/app/src/main/java/com/andrew/hdss/data/dtos/VisitPushDto.kt
package com.andrew.hdss.data.dtos

import com.andrew.hdss.data.models.enums.VisitStatus
import kotlinx.serialization.Serializable
import java.time.LocalDateTime

@Serializable
data class VisitPushDto(
    val clientId: String,
    val householdClientId: String,
    val visitDate: String,
    val status: VisitStatus
)
